lib/index.js.flow

```diff
@@ -0,0 +1,69 @@
+/* @flow */
+type ConnectAll = <D, P, S, C: React$Component<D, P, S>, SP, DP, Dispatch: Function>(
+  mapStateToProps: (state: Object) => SP,
+  mapDispatchToProps: (dispatch: Dispatch) => DP,
```

Comment:
An important case that we want to support is when `mapDispatchToProps` is an object with action creators (`{ addTodo, removeTodo }`) rather than a function. Then injected props will be functions with the same names (`{ addTodo, removeTodo }`) but those functions would dispatch automatically.